backend/src/main/kotlin/org/loculus/backend/controller/DevDashboardController.kt

```diff
@@ -0,0 +1,17 @@
+package org.loculus.backend.controller
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.loculus.backend.service.submission.SubmissionDatabaseService
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/admin")
+@SecurityRequirement(name = "bearerAuth")
+class DevDashboardController(private val submissionDatabaseService: SubmissionDatabaseService) {
+    @Operation(summary = "Get number of processed sequence entries per pipeline version and organism")
+    @GetMapping("/pipeline-stats")
```

Comment:
I would prefer avoiding abbreviations and writing `/pipeline-statistics` as I experienced that abbreviations that are very obvious to me are not clear to others and writing them out improves clarity.

Reply:
👍